package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "GCWeb",
-  "version": "4.0.28-development",
```

Comment:
Would it be more appropriate to update this line in a separate PR? It probably should've happened as a part of #1352, but was probably missed.

Reply:
Yes it was missed, I don't think there is a value to have a separate PR.

Reply:
In the end I decided to bring this in via a separate PR (#1353). I'll remove that change from this PR.